tools/pubsys/Infra.toml.example

```diff
@@ -27,6 +27,17 @@ signing_keys = { default = { file = { path = "/home/user/key.pem" } } }
 # metadata_base_url = "https://example.com/metadata/"
 # targets_url = "https://example.com/targets/"
 
+# Arches and variants you publish to this repo.  This allows you to use the repo validation targets in the makefile.
+# arches = [ "x86_64", "aarch64" ]
+# variants = [
+#    "aws-k8s-1.15",
+#    "aws-k8s-1.16",
+#    "aws-k8s-1.17",
+#    "aws-ecs-1",
+# ]
+# Repository URLs are expected to match the structure in the `build/repos` directories:
+#     ${metadata_base_url}/${variant}/${arch}/
```

Comment:
What if we create a new table named `[existing-repo.NAME]` to house the arch variant combinations? This could also be a separate config file (Repos.toml?) from Infra.toml but would still be collated when calling pubsys validation subcommands in cargo make.

Something like:
```
[existing-repo.2020-02-02]
// These existing repositories are created using this repository infrastructure from Infra.toml
repo_infra = "repo.2020-02-02"
// The valid arch variant combinations
arches = [ "x86_64", "aarch64" ]
variants = [
    "aws-k8s-1.15",
    "aws-k8s-1.16",
    "aws-k8s-1.17",
    "aws-ecs-1",
]

[existing-repo.raspberry-pi]
repo_infra = "repo.${some_key}"
arches = ["aarch64"]
variants = [ "rpi-dev" ]
```

Reply:
> The idiomatic way to check multiple variants and architectures would be to loop over `cargo make` invocations, modifying `BUILDSYS_VARIANT` and `BUILDSYS_ARCH` as appropriate.

Your concern seems valid. The issue then becomes, where should the list of varian and arches to loop over be stored?

Reply:
> where should the list of variants and arches to loop over be stored?

There was some previous discussion of using a file in the repo bucket itself to enumerate what lives within.  Getting the format right could be tricky, but it would separate concerns.